src/Compilers/Core/Portable/SourceGeneration/Nodes/SyntaxReceiverStrategy.cs

```diff
@@ -40,7 +39,6 @@ private sealed class Builder : ISyntaxInputBuilder
 
             public Builder(SyntaxReceiverStrategy<T> owner, object key, StateTableStore driverStateTable, bool trackIncrementalSteps)
```

Comment:
Remove `owner` from the ctor as well?

Reply:
We can't. The `owner` parameter is still used in the body of the constructor.